matching-notification-api/src/main/java/org/phenotips/matchingnotification/match/internal/DefaultPatientInMatch.java

```diff
@@ -258,25 +260,10 @@ public boolean isLocal()
     @Override
     public Collection<String> getEmails()
     {
-        Collection<String> emails = new LinkedList<>();
-        if (this.isLocal()) {
-            // Note: The patient is not saved because sometimes (often?) this method is run not on an object that
-            // was just created, but on one created from a column from the DB. In this case, we might not have
-            // a Patient object when the patient is remote.
-            emails.addAll(this.getPatientEmails(this.patient));
-        } else {
-            if (StringUtils.isNotEmpty(this.href)) {
-                // TODO: discuss what better algorithm/package can be use to split emails
-                // MME emails may be of the form "mailto:email1,email2", need ot parse that
-                if (this.href.startsWith("mailto:")) {
-                    String emailList = this.href.replace("mailto:", "");
-                    emails.addAll(Arrays.asList(emailList.split(",|;")));
-                } else {
-                    emails.add(this.href);
-                }
-            }
+        if (this.contactInfo != null) {
+            return this.contactInfo.getEmails();
         }
-        return emails;
+        return null;
```

Comment:
there is a change in functionality, it used to return an empty collection when there were no emails, now it returns `null`. Null often requires an extra check in the calling code, any reason not to return an empty collection here as before?